test/compress/issue-3768.js

```diff
@@ -9,9 +9,9 @@ mangle: {
         })();
     }
     expect: {
-        var o = eval, e = 42;
+        var e = eval, x = 42;
         (function() {
-            console.log(o("typeof x"));
+            console.log(e("typeof x"));
         })();
     }
     expect_stdout: "undefined"
```

Comment:
This is minor, but could you change that line to:
```
     expect_stdout: true
```
to better reflect the result is different in a browser - namely "number".

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/3776/commits/a7a615658b59c8ec721d511363681d10643648d8